OKTableViewLiaison/Classes/Views/OKPaginationTableViewCell.swift

```diff
@@ -7,7 +7,7 @@
 
 import UIKit
 
-public final class OKPaginationTableViewCell: UITableViewCell {
+final public class OKPaginationTableViewCell: UITableViewCell {
```

Comment:
I think it made more sense the other way around. The class is final, whether or not it is public is the modifier. So a `public final class`